Modules/@babylonjs/react-native-iosandroid/android/src/main/java/com/babylonreactnative/EngineView.java

```diff
@@ -26,17 +27,19 @@
 
 public final class EngineView extends FrameLayout implements SurfaceHolder.Callback, TextureView.SurfaceTextureListener, View.OnTouchListener {
     private static final FrameLayout.LayoutParams childViewLayoutParams = new FrameLayout.LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.MATCH_PARENT);
-    private TextureView transparentTextureView;
-    private Surface transparentSurface = null;
-    private SurfaceView opaqueSurfaceView = null;
+    private SurfaceView surfaceView = null;
```

Comment:
To me the names were more clear the way they were before (`opaqueSurfaceView` and `transparentTextureView`). Unless you object will you change them back (and restore the original order to simplify the diff for this PR)?

Reply:
I can make it more clear for the textureView to rename it transparentTextureView. But the surfaceView can be opaque or transparent.

Reply:
Ok, maybe if they just stay in the same order then it will be more clear.